src/styles/landing.module.scss

```diff
@@ -0,0 +1,116 @@
+@mixin centerStuff {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+@mixin border_outline() {
+  border: none;
+  outline: none;
+}
```

Comment:
please create `mixin` file sams as variables and move all mixins there. 

**If these mixins not get parameters consider using placeholders it's might be better option**